features/_your_details/your_details_editing.feature

```diff
@@ -1,21 +1,19 @@
-Feature:
-  As a user of the WPCC tool,
+Feature: Your Details editing
   In order to change my details,
-  I want to be able to go back to the first step.
+  As a user of the WPCC tool,
+  I want to be able to return to edit my choices.
 
 Background:
-  Given I am on the WPCC homepage
+  Given I have valid details
 
 Scenario: on Your Contributions page
-  And   I complete the your details form and move to your contributions page
-  When  I click edit on the your details summary
-  Then  I should return to the your details page
-  And   I should see my details in the form fields
+  And I am on Your Contributions step of the tool
```

Comment:
is the standard that we dont align the spacing of Given, And, When, Then. I don't have a preference either way, just want to make sure we're all on the same page.